jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java

```diff
@@ -202,7 +202,7 @@ public void customize_withFromTargetDirConfiguredAndNoCmd_shouldNotInitializeInv
         .hasFieldOrPropertyWithValue("env", Collections.singletonMap("DEPLOY_DIR", "/usr/local/tomcat/webapps"))
         .hasFieldOrPropertyWithValue("cmd", null);
   }
-
+  
```

Comment:
plz remove this change :)

Reply:
I tried hard reset that file, but it remains on this PR for some reason:/

Reply:
it's because you are using merge rather than rebase when updating the PR from master which makes things hard to fix

Reply:
Having that you remotes are named like that:
```
$ git remote -v
origin	https://github.com/Prathamkrishna/jkube (fetch)
origin	https://github.com/Prathamkrishna/jkube (push)
upstream	https://github.com/eclipse/jkube (fetch)
upstream	https://github.com/eclipse/jkube (push
```

```
git log
```
find the right commit id of your commit: should be `24b955ac3fafa9682d06bf8c06f15bcc54059472`
```
git checkout \#1328
git reset --hard 24b955ac3fafa9682d06bf8c06f15bcc54059472
```
here we don't want the changes in the file `jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java` so let's remove it from the commit
```
git checkout HEAD~1 jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java
git commit --amend
```
and finally
```
git rebase upstream/master
git push origin  \#1328 --force-with-lease
```

Reply:
Hey, alright I think I've resolved a bit. there were a few incoming changes in the WebAppGeneratorTest.java file so I've added those changes and pushed them on the branch

Reply:
it's still there